piet-scene/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "piet-scene"
+version = "0.1.0"
+license = "MIT/Apache-2.0"
+edition = "2021"
+
+[dependencies]
+bytemuck = { version = "1.7.2", features = ["derive"] }
+smallvec = "1.8.0"
+pinot = "0.1.5"
+moscato = "0.1.2"
+
+# remove these and move demo to another directory
```

Comment:
Should this comment be labeled "TODO"?

Reply:
Oops. Good catch. I'm happy to just remove the binary altogether along with its dependencies. I believe we discussed adding a more structured example framework to the project and it seems like "soon" might be a good time to do that.